src/test/java/com/openlattice/shuttle/conditions/ConditionTest.java

```diff
@@ -54,7 +54,7 @@ public void testCompareCondition() {
         Map<String, String> row = getTestRow();
         String[] datePattern = { "MM/dd/yyyy HH:mm" };
         ArrayList<Transformation> transforms = new ArrayList<>();
-        transforms.add( new DateTimeTransform( datePattern ) );
+        transforms.add( new DateTimeTransform( datePattern, null ) );
```

Comment:
if DateTimeTransform will frequently be initialized without a timezone it's probably cleaner to just create a second constructor with only the pattern parameter

Reply:
you can remove the null parameters now (seems like the tests shouldn't pass with them?)